addons/account/models/account_move.py

```diff
@@ -2121,12 +2121,12 @@ def _check_balanced(self, container):
                 move = self.browse(move_id)
                 error_msg += _(
                     "\n\n"
-                    "The move (%(move)s) is not balanced.\n"
-                    "The total of debits equals %(debit_total)s and the total of credits equals %(credit_total)s.\n"
-                    "You might want to specify a default account on journal \"%(journal)s\" to automatically balance each move.",
+                    "Unbalanced move alert! The move (%(move)s) is feeling a bit unbalanced,"
```

Comment:
Replace "Unbalanced move alert! The move (%(move)s) is feeling a bit unbalanced,"
by "The journal entry is not balanced,"